packages/commerce-sdk-react/CHANGELOG.md

```diff
@@ -1,3 +1,4 @@
+## v2.7.4 (Aug 28, 2023)
 ## v2.7.4-preview.0 (Jul 31, 2023)
```

Comment:
```suggestion
```
Not super important here, but when doing full releases we should ditch the headings for the preview releases and merge the notes into a single heading for the full release.

Reply:
eyy, sniped

Reply:
It has been done